src/app/3d/qgs3dmapconfigwidget.cpp

```diff
@@ -161,6 +161,31 @@ Qgs3DMapConfigWidget::Qgs3DMapConfigWidget( Qgs3DMapSettings *map, QgsMapCanvas
   connect( widgetLights, &QgsLightsWidget::lightsRemoved, this, &Qgs3DMapConfigWidget::validate );
 
   groupShadowRendering->setChecked( map->shadowSettings().renderShadows() );
+
+  edlGroupBox->setChecked( map->eyeDomeLightingEnabled() );
+  edlStrengthSpinBox->setValue( map->eyeDomeLightingStrength() );
+  edlDistanceSpinBox->setValue( map->eyeDomeLightingDistance() );
+
+  mDebugShadowMapCornerComboBox->addItem( QStringLiteral( "Top Right" ) );
+  mDebugShadowMapCornerComboBox->addItem( QStringLiteral( "Top Left" ) );
+  mDebugShadowMapCornerComboBox->addItem( QStringLiteral( "Bottom Left" ) );
+  mDebugShadowMapCornerComboBox->addItem( QStringLiteral( "Bottom Right" ) );
+
+  mDebugDepthMapCornerComboBox->addItem( QStringLiteral( "Top Right" ) );
+  mDebugDepthMapCornerComboBox->addItem( QStringLiteral( "Top Left" ) );
+  mDebugDepthMapCornerComboBox->addItem( QStringLiteral( "Bottom Left" ) );
+  mDebugDepthMapCornerComboBox->addItem( QStringLiteral( "Bottom Right" ) );
```

Comment:
all these strings should be translatable - i.e. wrapped in tr()

also best to avoid code duplication...